8.cytopick_analysis/notebooks/find_prediction_images.ipynb

```diff
@@ -1,1914 +1,1911 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    def top_n_cell_select(
</pre><p>The function name does not reflect what you have documented. Consider updating the function name if it is not a big of a change. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>